docs/dashboards/data-quality-dashboard.asciidoc

```diff
@@ -0,0 +1,59 @@
+[[data-quality-dash]]
+= Data Quality dashboard
+
+The Data Quality dashboard shows you whether your data is effectively mapped to the https://www.elastic.co/guide/en/ecs/current/ecs-reference.html[Elastic Common Schema] (ECS). Successful {ref}/mapping.html[mapping] enables you to search, visualize, and interact with your data throughout {es-sec} and {kib}.
+
+image::images/data-qual-dash.png[The Data Quality dashboard]
+
+You can use the Data Quality dashboard to:
+
+* Check one or multiple indices (by default, the indices used by {es-sec}) for unsuccessful mappings, to help you identify problems.
+* View detailed information about the fields in checked indices.
+* Track unsuccessful mappings by creating a case or markdown report based on Data Quality results.
+
+NOTE: The Data Quality dashboard does not show data from cold or frozen {ref}/data-tiers.html[data tiers]. It also does not display data from remote clusters using cross-cluster search — to view data from another cluster, log in to that cluster's {kib} instance.
+
+.Requirements
+[sidebar]
+--
+To use the Data Quality dashboard, you need at least the following {ref}/security-privileges.html#privileges-list-indices[permissions] for each index you want to check:
+
+. `monitor` or `manage` (required for the {ref}/indices-stats.html[Index stats API])
+. `view_index_metadata` or `manage_ilm` (required for the {ref}/ilm-explain-lifecycle.html[Explain lifecycle API])
+. `view_index_metadata` or `manage` (required for the {ref}/indices-get-mapping.html[Get mapping API])
+. `read` (required for the {ref}/search-search.html[Search API])
+--
+
+[discrete]
+== Check indices
+When you open the dashboard, data does not appear until you select which indices to check.
+
+*Check multiple indices:*
+To check all indices in the current data view, click *Check all* at the top of the dashboard. A progress indicator will appear.
+
+*Check a single index:*
+To check a single index, expand it using the arrow on the left. Checking a single index is faster than checking all indices.
+
+[discrete]
+== View detailed information about the fields in a checked index
```

Comment:
You can prob shorten this since you explain it in the following sentence. Also, if you want to remove "view" from the title, here's another alternative:

```suggestion
== Learn more about checked index fields
```